from unittest import TestCase


class Test_main(TestCase):
    def test_main(self):
        self.fail()
